app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,5 +1,25 @@
+import dayjs from "dayjs";
+import * as yup from "yup";
+
 import { renderContactDetails, renderText } from "components/Dashboard/utils";
 
+export const CONTACTS_FORM_INITIAL_FORM_VALUES = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  role: "",
```

Comment:
`role` is said to be an `object` in your validation schema. But you've provided an empty string as its initial value. It doesn't make sense to me. Please make this semantically correct

Reply:
@yedhink I couldn't make the validation work after adding the shape of the object. Can I assign default values from the `role select component` to the `CONTACTS_FORM_INITIAL_FORM_VALUES`  or is there something I have missed?

<img width="887" alt="Screenshot 2023-02-09 at 9 17 47 PM" src="https://user-images.githubusercontent.com/35962711/217863688-044456ab-7d54-49f3-a7b5-bab43934d96e.png">

Reply:
@jenipharachel 

>Can I assign default values from the role select component to the CONTACTS_FORM_INITIAL_FORM_VALUES

That's not a good solution. Because we can't always know what the backend API's `role` would be shaped with. We can only know a generic shape, not a specific one.

>is there something I have missed?

I want you to put up a video and explain to me your understanding of what `nullable` means in `yup`. Because once you understand it, you should be able to fix the issue here.

Reply:
@yedhink https://youtu.be/M-yYaef87uc

Reply:
LGTM. That's the correct solution.